### PR TITLE
feat(Column): add column component

### PR DESCRIPTION
**Pull request by arwehrman**
This PR adds Column component

-  Adds Column component using `Column` from Solid primitives
-  Adds `useFocusManager` to decorator
- Removes width from `Args` in `Preview`. Placing `width` in args of Preview will add the `width`  control to every Story. We could override this by disabling the control in the argTypes of each Story that you don't want the `width` to show in but that seems excessive.

### CODE REVIEW

File: src/packages/solid/components/Column/Column.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { Component, createEffect, onMount, on } from 'solid-js';
```
Comment (chiefcll) [inline src/packages/solid/components/Column/Column.tsx:1]: I think Column can just be alias to https://github.com/lightning-js/solid-primitives/blob/main/packages/Column.tsx

Or we should discuss combining them but I don't think the default column needs to do anything special

Comment (arwehrman) [reply src/packages/solid/components/Column/Column.tsx:1]: So we don't need to build out a Column component or story? We are just going to import Column from primitives when we need to use it in other components?

Comment (anthony9187) [reply src/packages/solid/components/Column/Column.tsx:1]: the LUI Column handles the position of the scrolling a little differently - the focused content remains centered in the screen and the other column items scroll up through it. is this something we'd want to pull into the primitive?

File: src/packages/solid/.storybook/preview.tsx [hunk 2]
```diff
@@ -35,9 +36,6 @@ export const argTypes = {
     description: 'the width of the component'
   }
 };
-export const args = {
```
Comment (arwehrman) [inline src/packages/solid/.storybook/preview.tsx:38]: See comment in PR description on why I removed this.

File: src/packages/solid/.storybook/preview.tsx [hunk 3]
```diff
@@ -55,6 +53,11 @@ const preview = {
   },
   decorators: [
     (Story) => {
+      useFocusManager({
+        Menu: 'm',
```
Comment (chiefcll) [inline src/packages/solid/.storybook/preview.tsx:57]: you don't need to pass any key mapping it - it does defaults for left, right up down enter...

**Comment by chiefcll** [comment]
@anthony9187 I'd prefer not to have two column components. If UI-components wants to own both row and column from primitives. I'm not sure what this column component does that is different or needs to be different..

**Comment by chiefcll** [comment]
@anthony9187 I'd prefer not to have two column components. If UI-components wants to own both row and column from primitives. I'm not sure what this column component does that is different or needs to be different..

**Review by anthony9187 — APPROVED** [review]
we should probably make width and height configurable via style files, but we can revisit that later

**Pull request merged**